ietf/templates/meeting/proceedings.html

```diff
@@ -147,6 +147,36 @@ <h2 class="mt-5" id="iab">
             </tbody>
         </table>
     {% endif %}
+    <!-- Editorial Sessions -->
```

Comment:
This should go after irtf

Reply:
Fixed with 41aff3d42c62fd356c16fe42277b394ac6cdbb87